client/charts/js/components/IncidentsTimeBarChart.jsx

```diff
@@ -38,7 +38,7 @@ export default function IncidentsTimeBarChart({
 
 		if (groupByTag) {
 			// if groupByTag is set, that means we are filtering a tag
-			if (d.tags.indexOf(groupByTag) >= 0) {
+			if (d.tags?.indexOf(groupByTag) >= 0) {
```

Comment:
Didn't notice this when it was first committed but I love the optional chaining operator!

Reply:
So nice and concise!